### ISSUE TITLE
Empty attackmate.yml does not work

### ISSUE THREAD

**Issue by whotwagner** [ait-testbed/attackmate#73]
Attackmate does not work with an empty attackmate.yml

**Comment by whotwagner** [comment]
should be fixed in development-branch

**Comment by whotwagner** [comment]
not fixed:

```
Traceback (most recent call last):
  File "/home/kali/attackmate/venv/bin/attackmate", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/kali/attackmate/venv/lib/python3.11/site-packages/attackmate/__main__.py", line 163, in main
    hacky = AttackMate(parse_playbook(args.playbook, logger), parse_config(args.config, logger))
                                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/attackmate/venv/lib/python3.11/site-packages/attackmate/__main__.py", line 90, in parse_config
    cfg = load_configfile(file)
          ^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/attackmate/venv/lib/python3.11/site-packages/attackmate/__main__.py", line 59, in load_configfile
    return Config.parse_obj(config)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/attackmate/venv/lib/python3.11/site-packages/pydantic/main.py", line 1118, in parse_obj
    return cls.model_validate(obj)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/attackmate/venv/lib/python3.11/site-packages/pydantic/main.py", line 551, in model_validate
    return cls.__pydantic_validator__.validate_python(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.ValidationError: 1 validation error for Config
  Input should be a valid dictionary or instance of Config [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.7/v/model_type
```

**Comment by annaerdi** [comment]
Fixed in PR #88
